lua/telescope/_extensions/file_browser/picker.lua

```diff
@@ -73,6 +73,7 @@ local fb_picker = {}
 ---@field display_stat boolean|table: ordered stat; see above notes, (default: `{ date = true, size = true }`)
 ---@field hijack_netrw boolean: use telescope file browser when opening directory paths; must be set on `setup` (default: false)
 ---@field use_fd boolean: use `fd` if available over `plenary.scandir` (default: true)
+---@field git_status boolean: show the git status of files (default: false)
```

Comment:
I'd say we should probably enable this by default even.

In my RFC PR the auto switcher to `find_files` like mode will turn this off then for speed.

Reply:
Sweet, I'm always a bit afraid of people starting to complain when their default behaviour changes :D I definitely would want this turned on!